src/server/req_runjob.c

```diff
@@ -985,11 +985,15 @@ svr_strtjob2(job *pjob, struct batch_request *preq)
 
 	/* send the job to MOM */
 
+	/* clearing old comment since we are sending a fresh job */
+	job_attr_def[(int) JOB_ATR_Comment].at_decode(
+			&pjob->ji_wattr[(int) JOB_ATR_Comment], (char *) 0,
+			(char *) 0, "");
```

Comment:
I'm confused on this as well.  Why clear the comment if you are just going to set it again down the line?